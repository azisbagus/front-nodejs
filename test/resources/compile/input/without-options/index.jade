.container
    header
        h1 Steer the Beat

    footer
        p Copyright &copy; 2012 Steer the Beat.
