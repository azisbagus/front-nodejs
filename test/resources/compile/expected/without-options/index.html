<div class="container"><header><h1>Steer the Beat</h1></header><footer><p>Copyright &copy; 2012 Steer the Beat.</p></footer></div>